Makefile

```diff
@@ -66,7 +63,8 @@ test-e2e-local: setup
 
 push-to-minikube-image-registry:
 	docker run -d -p 5000:5000 --restart=always --name image-registry registry || true
-	DOCKER_REPO="localhost:5000/" IMAGE="wildfly-operator" make push
+	docker build -t "localhost:5000/$(IMAGE):$(TAG)" . -f build/Dockerfile
```

Comment:
that'd be better if we could use the multiarch for testing.
Would something like that work?

```
push-to-minikube-image-registry:
 	docker run -d -p 5000:5000 --restart=always --name image-registry registry || true
        DOCKER_REPO="localhost:5000/" make push
```